File: Code Chef/JavaScript/400-499/434 Speciality SPECIALITY.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    
    return intArray;
};

const checkUserSpeciality = (problemsSet, problemsTested, problemsWritten) => {
    const userSpeciality = Math.max(problemsSet, problemsTested, problemsWritten);
    
    (userSpeciality === problemsSet)
        ? console.log("Setter")
        : (userSpeciality === problemsTested)
            ? console.log("Tester")
            : console.log("Editorialist");
}

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n"));
    
    testcases.forEach((testcase) => {
        const [problemsSet, problemsTested, problemsWritten] = testcase;
        checkUserSpeciality(problemsSet, problemsTested, problemsWritten);
    });
});

File: Code Chef/C Programming/400-499/434 Speciality SPECIALITY.c
#include <stdio.h>
int main(void) 
{
	int t,x,y,z;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d%d%d",&x,&y,&z);
	    if((x>y)&&(x>z))
	    {
	        printf("Setter\n");
	    }
	    else if((y>x)&&(y>z))
	    {
	        printf("Tester\n");
	    }
	    else if((z>y)&&(z>x))
	    {
	        printf("Editorialist\n");
	    }
	    
	}
	return 0;
}

